Npgsql.all.sln

```diff
@@ -22,16 +22,19 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.Benchmarks", "test\Npgsql.Benchmarks\Npgsql.Benchmarks.csproj", "{8B4AE9B6-CDAC-44DD-A5CD-28A470D363B8}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VSIX", "src\VSIX\VSIX.csproj", "{ECFD8615-DEAD-4498-B262-85A4D0230229}"
+	ProjectSection(ProjectDependencies) = postProject
+		{9D13B739-62B1-4190-B386-7A9547304EB3} = {9D13B739-62B1-4190-B386-7A9547304EB3}
+	EndProjectSection
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.Json.NET", "src\Npgsql.Json.NET\Npgsql.Json.NET.csproj", "{9CBE603F-6746-411D-A5FD-CB2C948CD7D0}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.NodaTime", "src\Npgsql.NodaTime\Npgsql.NodaTime.csproj", "{D8DF12D6-FA70-4653-BD8F-C188944836DE}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.PluginTests", "test\Npgsql.PluginTests\Npgsql.PluginTests.csproj", "{9BD7FC3D-6956-42A8-A586-2558C499EBA2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Npgsql.LegacyPostgis", "src\Npgsql.LegacyPostgis\Npgsql.LegacyPostgis.csproj", "{EC0DCB3C-9401-47BB-A5E8-B8C7A47DF96A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.LegacyPostgis", "src\Npgsql.LegacyPostgis\Npgsql.LegacyPostgis.csproj", "{EC0DCB3C-9401-47BB-A5E8-B8C7A47DF96A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Npgsql.RawPostgis", "src\Npgsql.RawPostgis\Npgsql.RawPostgis.csproj", "{B7E92398-DD4E-410E-923C-E256992F6687}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.RawPostgis", "src\Npgsql.RawPostgis\Npgsql.RawPostgis.csproj", "{B7E92398-DD4E-410E-923C-E256992F6687}"
```

Comment:
The identifiers of the `Npgsql.RawPostgis` and `Npgsql.LegacyPostgis` projects cross the ID of `Npgsql.Benchmarks` project. You can change them back by editing the solution file in any text editor.

Reply:
Thank you for comment and sorry for late reply.  
I have a question about this point. 
Why the identifiers of the Npgsql.RawPostgis and Npgsql.LegacyPostgis should be the same as VSIX's one (FAE04EC0-301F-11D3-BF4B-00C04F79EFBC)? 
In my understanding, Npgsql.Json.NET is alos plugin of Npgsql and the identifiers of Npgsql.Json.NET is the same as Npgsql's one (9A19103F-16F7-4668-BE54-9A1E7A4F7556).
I think it is better that the identifiers of the Npgsql.RawPostgis and Npgsql.LegacyPostgis should be the same as Npgsql's one. 
I do not have much knowledge about solution file, please let me know if my understanding is wrong.

And even if I fix this point as you suggest, the identifiers are back to FAE04EC0-301F-11D3-BF4B-00C04F79EFBC when I build this solution.

Reply:
They should be unique for each project. You can read more about in on [MSDN](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file). I don't know why the IDE changes identifiers, but you can omit that kind of changes when making a commit. Just stage files which you have changed to fix the issue.

Reply:
Even if I omit this change at this PR, it is hard for VSIX developer and user like me. 
Because I will have to omit this every time when I try to submit PR. 
So, I revert GUIDs changes and the cause of this (Project dependencies). 
However this PR complete compiling VSIX project still now.

Reply:
Everything was okay because the first identifier is a project type, so it could be duplicated. The second one is a project identifier which must be unique. Sorry that misled you.

The reason why the identifiers were changed is because of a two year old commit to `dotnet/project-system`. It changed the C# project type identifier from `FAE04EC0-301F-11d3-BF4B-00C04F79EFBC` to `9A19103F-16F7-4668-BE54-9A1E7A4F7556`.

Reply:
If you would like to update type identifiers you can do it for `Npgsql.sln` too.

Thanks for your contribution!

Reply:
Thank you for comment. I will update identifiers on another PR.